priv/repo/migrations/20210923231044_remove_status_non_null_from_hotspots.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Console.Repo.Migrations.RemoveStatusNonNullFromHotspots do
+  use Ecto.Migration
+
+  def change do
+    alter table(:hotspots) do
+      modify(:status, :string, null: true)
```

Comment:
noticed that we can get status of null from blockchain api, and upon encountering said row, will stop the job completely